### ISSUE TITLE
Configuration signature for `t.field`

### ISSUE THREAD

**Issue by jasonkuhrt** [graphql-nexus/nexus#920]
Consider the new Nexus Prisma API: https://github.com/prisma/nexus-prisma#usage

I would like to overload `t.field` with a new signature so that instead of:

```ts
export const schema = makeSchema({
  types: [
    objectType({
      name: User.$name
      description: User.$description
      definition(t) {
        t.field(User.id.name, User.id)
      }
    })
  ]
})
```

We can do this:

```ts
export const schema = makeSchema({
  types: [
    objectType({
      name: User.$name
      description: User.$description
      definition(t) {
        t.field(User.id)
      }
    })
  ]
})
```

**Comment by jasonkuhrt** [comment]
I discussed this feature with @tgriesser over Slack some weeks ago and there seemed to be no major objections.

**Comment by jasonkuhrt** [comment]
closed by #938
